website/content/docs/security/acl/acl-rules.mdx

```diff
@@ -297,7 +394,86 @@ keyring = "write"
 There's only one keyring policy allowed per rule set, and its value is set to one of the policy
 dispositions. In the example above, the keyring may be read and updated.
 
-#### Node Rules
+### Mesh Rules
+
+The `mesh` resource controls access to ingress gateways, terminating gateways, and mesh configuration entries. 
+
+See [Admin Partition Rules](#admin-partition-rules) for an example rule that uses the `mesh` resource. 
+ 
+
+### Namespace Rules <EnterpriseAlert inline />
+
+The `namespace` and `namespace_prefix` resource controls access to Consul namespaces. Namespaces define a scope of resources for which ACL rules apply. ACL rules, themselves, can then be defined to only to apply to specific namespaces. 
+
+-> **Consul 1.7.0 and later**: The ability to add many types of resources to separate namespaces was added to [Consul Enterprise](https://www.hashicorp.com/consul) 1.7.0. 
+
+The following examples describe how namespace rules can be defined in a policy:
+
+```hcl
+namespace_prefix "" {
+
```

Comment:
I think it'd be good to include an example here in the namespace_prefix as well, since I expect that's the more common use case for the namespace policy line.

```
# grants permission to create and edit all namespace
policy = "write"
```